### PR TITLE
🌱 Bump CAPI to 1.6.6 in api and test folders

### PR DESCRIPTION
**Pull request by adilGhaffarDev**
**What this PR does / why we need it**:
Bump CAPI to 1.6.6 in api and test folders

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,8 +45,8 @@ need to be manually installed. Example flow of installing Metal3 provider:
    level of the logging verbose with a positive integer number, ie. -v5.
 
    ```shell
-   clusterctl init --core cluster-api:v1.6.5 --bootstrap kubeadm:v1.6.5 \
-       --control-plane kubeadm:v1.6.5 -v5
+   clusterctl init --core cluster-api:v1.6.6 --bootstrap kubeadm:v1.6.6 \
+       --control-plane kubeadm:v1.6.6 -v5
```
Comment (kashifest) [inline README.md:49]: @adilGhaffarDev can you also update line 63 ? 

`clusterctl init --infrastructure metal3:v1.5.0`

Comment (adilGhaffarDev) [reply README.md:49]: done

**Review by Rozzii — COMMENTED** [review]
/lgtm

**Pull request merged**